### DISCUSSION TITLE
discord RPC option?

### DISCUSSION THREAD

**Discussion by Faizan11602070** [thatskymod/Sky-CotL-Scripts#147] [Q&A]
what is the discord RPC option?

**Comment by thatskymod** [comment]
In computer science, **RPC** stands for [Remote Procedure Call](https://www.google.com/search?q=Remote+Procedure+Call). That's a protocol which enables applications to communicate over a network as though they were on the same system.

On Discord, however, **RPC** refers to Rich Presence. Rich Presence allows users to share live information about their current activity on their profile. This might be something like "playing [game name] for [n] hours", the level you're on, or the mode you're playing, often accompanied by custom icons or colors.

You can learn more about Discord's Rich Presence on their developer portal: https://discord.com/developers/docs/rich-presence/overview.

I can't tell how this is implemented in Sky mods specifically, or if it works as expected, but this should give you a general sense of what the option represents.
